src/Passes/examples/RandomWalkPE3/QSharpVersion/Example.qs

```diff
@@ -0,0 +1,78 @@
+﻿namespace Microsoft.Quantum.Qir.Emission {
+
+    open Microsoft.Quantum.Intrinsic;
+    open Microsoft.Quantum.Math;
+    open Microsoft.Quantum.Measurement;
+
+    operation CRz (theta : Double, control : Qubit, target : Qubit) : Unit {
+        body intrinsic;
+    }
+
+    operation Prepare(target : Qubit) : Unit {
+        H(target);
+    }
+
+    operation Iterate(c1 : Double, c2 : Double, target : Qubit, aux : Qubit) : Result {
+        within {
+            H(aux);
+        } apply {
+            Rz(c1, aux);
+            CRz(c2, aux, target);
+        }
+        return MResetZ(aux);
+    }
+
+    function ClassicalComputeInput1(mu : Double, sigma : Double) : Double {
+        return (PI() / 2.0) - (mu / sigma);
+    }
+
+    function ClassicalComputeInput2(mu : Double, sigma : Double) : Double {
+        return PI() * (mu - PI() * sigma / 2.0);
+    }
+
+    // function ClassicalCompute(mu : Double, sigma : Double) : (Double, Double) {

```

Comment:
These commented out functions were for my personal reference and don't need to be kept. Likewise with the calls to them below.

Reply:
Removed.